engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41700to41710.java

```diff
@@ -83,4 +96,25 @@ public void updateSystemVmTemplates(Connection conn) {
             throw new CloudRuntimeException("Failed to find / register SystemVM template(s)");
         }
     }
+
+    private void updateStorageType() {
+        storageDao = new PrimaryDataStoreDaoImpl();
+        List<StoragePoolVO> storPoolPools = storageDao.findPoolsByProvider("StorPool");
+        for (StoragePoolVO storagePoolVO : storPoolPools) {
+            if (StoragePoolType.SharedMountPoint == storagePoolVO.getPoolType()) {
+                storagePoolVO.setPoolType(StoragePoolType.StorPool);
+                storageDao.update(storagePoolVO.getId(), storagePoolVO);
+            }
+            updateStorPoolVolumesToStorPoolType(storagePoolVO.getId());
```

Comment:
better move this call to if above

Reply:
I don't think there will be cases with StorPool primary storage with a different storage type than the `SharedMountPoint`. But just in case, if there are any, I want to update all the volumes of the pool with the new storage type